frontend/src/components/shareProject/AddSummarySection.js

```diff
@@ -68,6 +68,7 @@ export default function AddSummarySection({
           value={projectData.short_description}
         />
       </div>
+      <Typography> {projectData.short_description ? projectData.short_description.length: 0} / 280</Typography>
```

Comment:
Make sure to not mix changes from different PR's in here.

Reply:
Thank you for your review i reverted the changes and it should be clean now.